app/models/kit_allocation.rb

```diff
@@ -1,17 +0,0 @@
-# == Schema Information
-#
-# Table name: kit_allocations
-#
-#  id                  :bigint           not null, primary key
-#  kit_allocation_type :enum             default("inventory_in"), not null
-#  created_at          :datetime         not null
-#  updated_at          :datetime         not null
-#  kit_id              :bigint           not null
-#  organization_id     :bigint           not null
-#  storage_location_id :bigint           not null
-#
-class KitAllocation < ApplicationRecord
```

Comment:
Not sure what this model was used for previously, but looks like it's currently unused.